src/scripts/scripts.js

```diff
@@ -0,0 +1,825 @@
+        en: 'right',
+        ru: 'right',
+        code: 'ArrowRight',
+      },
+    ],
+  },
+
+  init() {
+    this.elements.main = document.createElement('div');
+    this.elements.textarea = document.createElement('textarea');
+
+    this.elements.textarea.classList.add('keyboard-output');
+    this.elements.main.classList.add('keyboard');
+
+    if (localStorage.getItem('language')) {
+      this.properties.language = localStorage.getItem('language');
+    }
+
+    this.elements.main.appendChild(this.createKeyboard(this.properties.language));
+    this.elements.keys = this.elements.main.querySelectorAll('.keyboard__key');
+
+    document.body.appendChild(this.elements.textarea);
+    document.body.appendChild(this.elements.main);
+  },
+
+  createKeyboard(language) {
+    const fragment = document.createDocumentFragment();
+
+    this.properties.keyLayout.forEach((key, i) => {
+      const keyEl = document.createElement('button');
+      const insertBrAfter = [8, 220, 13, 38].indexOf(key.keyCode) !== -1;
+
+      keyEl.classList.add('keyboard__key');
+
+      document.addEventListener('keydown', () => {
+        if (i === 0) {
+          // console.log(event);
+          this.languageToggle(this.properties.language);
+        }
+      });
+
+      switch (key[language]) {
+        case 'backspace':
+          keyEl.textContent = key[language];
+          keyEl.classList.add('keyboard__key--size-2');
+
+          document.addEventListener('keydown', (event) => {
+            if (event.code === key.code) {
+              event.preventDefault();
+              keyEl.classList.add('keyboard__key--pressed');
+
+              this.properties.value = this.properties.value.slice(0, this.properties.value.length - 1);
+              this.onInput();
+            }
+          });
+
+          document.addEventListener('keyup', (event) => {
+            if (event.code === key.code) {
+              keyEl.classList.remove('keyboard__key--pressed');
+            }
+          });
+
+          keyEl.addEventListener('mousedown', () => {
+            keyEl.classList.add('keyboard__key--pressed');
+
+            this.properties.value = this.properties.value.slice(0, this.properties.value.length - 1);
+            this.onInput();
+          });
+
+          break;
+
+        case 'tab':
+          keyEl.textContent = key[language];
+          keyEl.classList.add('keyboard__key--size-2');
+
+          document.addEventListener('keydown', (event) => {
+            if (event.code === key.code) {
+              event.preventDefault();
+
+              keyEl.classList.add('keyboard__key--pressed');
+
+              this.properties.value += '\t';
+              this.onInput();
+            }
+          });
+
+          document.addEventListener('keyup', (event) => {
+            if (event.code === key.code) {
+              keyEl.classList.remove('keyboard__key--pressed');
+            }
+          });
+
+          keyEl.addEventListener('mousedown', () => {
```

Comment:
And it is better to use event delegation instead of creating listeners on each key